test/main/java/jp/ac/utokyo/rcast/karkinos/alleliccnv/CheckPossibleHDAmpTest.java

```diff
@@ -0,0 +1,53 @@
+package jp.ac.utokyo.rcast.karkinos.alleliccnv;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.MethodSource;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import jp.ac.utokyo.rcast.karkinos.exec.CapInterval;
+import jp.ac.utokyo.rcast.karkinos.exec.CopyNumberInterval;
+
+// TODO: This test class only aims at testing CopyNumberInterval and CapInterval.
+// We should add more tests in the future.
+public class CheckPossibleHDAmpTest {
+  private static final int unit = 1000000;
+
+  @Test
+  void in1mTest() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+    final Method in1m = CheckPossibleHDAmp.class.getDeclaredMethod("in1m",
+        CopyNumberInterval.class, CapInterval.class);
+    in1m.setAccessible(true);
+
+    CopyNumberInterval tmp_cni = new CopyNumberInterval();
+    tmp_cni.setStart(2 * unit + 4);
+    tmp_cni.setEnd(2 * unit + 6);
+    final CopyNumberInterval cni = tmp_cni;
```

Comment:
This will probably suffice:

```java
final CopyNumberInterval cni = new CopyNumberInterval();
cni.setStart(2 * unit + 4);
cni.setEnd(2 * unit + 6);
```